admin_manual/maintenance/update.rst

```diff
@@ -145,20 +145,49 @@ user is::
 
     chown -R <http-user>:<http-user> /path/to/owncloud/
 
-* This example is for Ubuntu 14.04 LTS server::
-   
-    chown -R www-data:www-data /var/www/owncloud
-
-* Arch Linux::
-
-    chown -R http:http /path/to/owncloud/
+You can find your HTTP user in your HTTP server configuration files. Or you can 
+use :ref:`label-phpinfo` (Look for the **User/Group** line).
+
+* The HTTP user and group in Debian/Ubuntu is ``www-data``.
+* The HTTP user and group in Fedora/CentOS is ``apache``.
+* The HTTP user and group in Arch Linux is ``http``.
+* The HTTP user in openSUSE is ``wwwrun``, and the HTTP group is ``www``.
+
+The easy way to set the correct permissions is to copy and run the following scripts.
+The first script sets the permissions for the update process, the second script
+resets the permissions back to the strict settings described above. 
+Replace the ``ocpath`` variable with the path to your ownCloud directory, and 
+replace the ``htuser`` and ``htgroup`` variables with your HTTP user and group.
+
+Script preparing the instance for upgrading
+-------------------------------------------
+  ::
+  
+    #!/bin/bash
+    # Sets permissions of the owncloud instance for updating
+    
+    ocpath='/var/www/owncloud'
+    htuser='www-data'
+    htgroup='www-data'
+    rootuser='root' # On QNAP this is admin
+    
+    # If the script from the Installation Wizard section:
+    # "Setting Strong Directory Permissions" has been used, you do not need to reset
+    # the permissions on /apps, /config, /data and /themes
+    # as they already should be correct (${htuser}:${htgroup})
```

Comment:
OK, thanks @mmattel, I understand now. Sometimes it takes me awhile. I'm merging this, and then I am going to make a few small tweaks. Nothing major, just smoothing out some wording. Thanks again, I appreciate your contributing.